internal/pkg/aws/s3/s3_test.go

```diff
@@ -250,7 +250,7 @@ func TestS3_EmptyBucket(t *testing.T) {
 
 			gotErr := service.EmptyBucket(tc.inBucket)
 
-			if gotErr != nil {
+			if tc.wantErr != nil {
```

Comment:
The test case already existed but it never tested properly because of the conditional in the test.